DetDescr/python/HcalGeometry.py

```diff
@@ -365,6 +396,23 @@ def make_v14(self):
             ecal_side_dy / 2.0,
         ]
 
+        self.v14.scint_length = [[back_hcal_scint_length for layer in range(back_hcal_numLayers)],
+                                 [0.] * side_hcal_numTotalLayers, # Filled below
+                                 [0.] * side_hcal_numTotalLayers,
+                                 [0.] * side_hcal_numTotalLayers,
+                                 [0.] * side_hcal_numTotalLayers]
+        for s in range(side_hcal_numSections):
```

Comment:
add comments on why section and layer index start at 1

Reply:
Yes!